webapp/app/model/eligibility_data.py

```diff
@@ -472,10 +472,10 @@ class ForeignCountryMaybeEligibility(RecursiveDataModel):
     That's because the ForeignCountry step is the last step of the flow and needs to decide which result page is
     displayed: 'success' or 'maybe'.
     """
-    has_no_other_income: Optional[OtherIncomeEligibilityData]
     foreign_country_eligibility: str
     user_a_has_elster_account_eligibility: str
     user_b_has_elster_account_eligibility: Optional[str]
+    has_no_other_income: Optional[OtherIncomeEligibilityData]
```

Comment:
Why did you change this?

Reply:
Change it back! Thought it should be last, but since there are no other previous steps, we should leave it on top.